foods.html

```diff
@@ -3,26 +3,19 @@
   <head>
     <meta charset="utf-8">
     <title>Quantified Self - Manage Foods</title>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src='./lib/foodSearch.js'></script>
```

Comment:
Nice, so it appears with this merge we'll be covering all preliminary changes I made to meals-dv before you branched off of it, as they had not been merge before. This particular addition in the foods.html file is specific for the in-html 'on-input:'searchFunction'' option within <input>. We may decide to do something different later, but for now I like this script tag here since it works entirely different than our other js.